packages/tupaia-web/src/theme/constants.ts

```diff
@@ -6,7 +6,3 @@
 /**
  * Theme constants that don't require being in the entire theme object, ie. things that won't change if a custom theme was applied
  */
-
-export const FONT_SIZES = {
-  viewTitle: '2em',
-};
```

Comment:
What are your thoughts on keeping font sizes standardised somewhere? So we don't end up hard coding them in different places?

Reply:
Yes good question. I guess I prefer to have a reusable component with styles encapsulated rather than having reusable styles separately. I find it quite hard to restyle and refactor components that are using global style settings. I also think that this particular heading is only going to be used in the auth modal so the styles for it can live in there. I do think that it would be good to have some reusable typography styles or components for typography styles that will be used widely. The one that comes to mind is the paragraph style. I'm not sure the best way to reuse that. Maybe the material-ui theme or maybe a specific <Text> component?

```
    color: white;
    font-size: 0.875rem;
    line-height: 1.3;